### PR TITLE
LOG-5998: fix max_line_bytes for audit logs

### PR DESCRIPTION
**Pull request by jcantrill**
### Description
This PR:
* bumps the max_line_bytes for audit logs
* bumps the max_read_line_bytes for audit logs

### Links
https://issues.redhat.com/browse/LOG-5998

### CODE REVIEW

File: internal/generator/vector/source/audit_logs.go [hunk 1]
```diff
@@ -13,7 +13,8 @@ include = ["/var/log/audit/audit.log"]
 host_key = "hostname"
 glob_minimum_cooldown_ms = 15000
 ignore_older_secs = 3600
-max_read_bytes = 3145728
+max_line_bytes = 3145728
+max_read_bytes =  262144
```
[the remark this answers is not in the document]
Comment (jcantrill) [reply internal/generator/vector/source/audit_logs.go:17]: agree.  in the docs the defaults identify read < max which should apply in our case too

[the remark this answers is not in the document]
Comment (jcantrill) [reply internal/generator/vector/source/audit_logs.go:17]: Actually, strike my last comment.  The "max_line_bytes" is essentially the largest length a line can be and the assumption is that when completely read, the result is a well formed line message (e.g. json).  max_read_bytes is the amount of bytes read in a "read" operation but doesn't say it has completely read a log entry

[the remark this answers is not in the document]
Comment (alanconway) [reply internal/generator/vector/source/audit_logs.go:17]: Agreed but under normal circumstances you could read a batch of lines that in total are longer than the longest acceptable single line.

[the remark this answers is not in the document]
Comment (cahartma) [reply internal/generator/vector/source/audit_logs.go:17]: Scanning the discord, it does seem that if not set to the same value, the "read" is approx 10x smaller in general  
ex.
` max_line_bytes: 16777216
max_read_bytes: 16777216
`
or 
` max_line_bytes = 1024000
max_read_bytes = 102400
`

[the remark this answers is not in the document]
Comment (jcantrill) [reply internal/generator/vector/source/audit_logs.go:17]: > Agreed but under normal circumstances you could read a batch of lines that in total are longer than the longest acceptable single line.

@alanconway these settings apply to a single stream of logs which in the audit case here or in the container log case is a single file.  `N reads * max_read_bytes <= max_line_bytes` for any one log stream

**Pull request merged**